src/client/CMakeLists.txt

```diff
@@ -2,8 +2,10 @@ set(TARGET_NAME client)
 set(LIB_NAME game_client_lib)
 
 set(FILES
+    camera.cpp
     client.cpp
     cube.cpp
+    main.cpp
```

Comment:
Didn't notice this at first, but this should be removed so that the unit tests don't break when including `game_client_lib`